lib/start-checking.js

```diff
@@ -6,12 +6,16 @@ internals.getOk = require('./get-ok')
 
 function startChecking (state, options) {
   options = parse(options)
-  handleInterval(state, options)
+  state !== undefined && state.method !== undefined && state.url !== undefined && handleInterval(state, options)
 }
 
 function handleInterval (state, options) {
   var timeout
   var ok = internals.getOk(state)
+  var timeoutHandler = function () {
```

Comment:
Now defined in module scope as suggested as a name function; is there any style guide that says that name functions are preferred, and if not, would it be good to add one?